### PR TITLE
Remove Component namespace on action

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Component/Action/PlaceHolderAction.php [hunk 1]
```diff
@@ -13,13 +13,10 @@
...
-final class PlaceHolderAction
-{
-    /**
-     * @param mixed $data
-     */
-    public function __invoke($data = null): mixed
+class_exists(\Sylius\Resource\Action\PlaceHolderAction::class);
+
+if (false) {
+    final class PlaceHolderAction extends \Sylius\Resource\Action\PlaceHolderAction
```
Comment (diimpp) [inline src/Component/Action/PlaceHolderAction.php]: Is it alright to extend `final` class?

**Review by diimpp — APPROVED** [review]

File: src/Component/Action/PlaceHolderAction.php (hunk 1 above)
Comment (loic425) [reply src/Component/Action/PlaceHolderAction.php]: Yes, it's strange but it's done like this. I already did that based on API Platform bc layer and it works.

**Review by lchrusciel — COMMENTED** [review]
I'm all in favour of this change about the namespace, but there is no point in doing a migration layer for something that didn't reach official release

**Comment by loic425** [comment]
> I'm all in favour of this change about the namespace, but there is no point in doing a migration layer for something that didn't reach official release

Indeed, but I prefer to use the same bc-layer for every moved classes/interfaces to avoid errors (even if they are new).

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, @loic425!

**Pull request merged**